core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1318,7 +1319,7 @@ public void run() {
               if (retryEnabled) {
                 throttle = getThrottle(config);
               }
-            } catch (RuntimeException re) {
+            } catch (MalformedConfigException re) {
```

Comment:
I don't think this is a safe change to make.  Previously, RuntimeException was pretty wide, and any error processing service config would be logged.   This makes gRPC resilient in case of unforeseen service config problems.  And able to be fixed if a bad service config was pushed.

Won't this make panic mode much more likely?